s3fs/core.py

```diff
@@ -1475,7 +1516,12 @@ def close(self):
         """
         if self.closed:
             return
-        self.cache = None
+
+        if self.file_backed_cache:
+            self.cache.close()
```

Comment:
If you just do `self.cache = None`, so the reference drops, doesn't the file get cleaned up anyway upon garbage collection?

Reply:
true.